src/import/animatron-importer.js

```diff
@@ -744,8 +748,16 @@ Import.fill = function(src) {
  */
 Import.stroke = function(src) {
     if (!src) return null;
-    return Brush.stroke(is.arr(src[1]) ? Import.grad(src[1])
-                                       : src[1], // paint
+    var fill;
+    if (is.str(src[0])) {
+        fill = src[0];
+    } else if (is.arr(src[0])) {
+        if (is.arr(src[0][0])) {
+            fill = Import.grad(src[0]);
+        }
+        fill = Import.pattern(src[0]);
+    }
+    return Brush.stroke(fill, // paint
```

Comment:
I mean, may be export format was changed or it was a bug here before, but then the line below looks improper.